### PR TITLE
feat: logbasemeasure

### PR DESCRIPTION
**Pull request by Nimrais**
This PRs add logbasemeasure function into API.

### CODE REVIEW

File: src/distributions/binomial.jl [hunk 1]
```diff
@@ -85,6 +85,7 @@ end
 isbasemeasureconstant(::Type{Binomial}) = NonConstantBaseMeasure()
 
 getbasemeasure(::Type{Binomial}, ntrials) = Base.Fix1(binomial, ntrials)
+getlogbasemeasure(::Type{Binomial}, ntrials) = (x) -> log(getbasemeasure(Binomial, ntrials)(x))
```
Comment (ismailsenoz) [inline src/distributions/binomial.jl:88]: This should be written with loggamma functions

Comment (Nimrais) [reply src/distributions/binomial.jl:88]: Yeah, you are right, thanks, also it should be done for Poisson. I did it for NegativeBinomial but, I was preoccupied with Arbitrary ef.

Comment (ismailsenoz) [reply src/distributions/binomial.jl]: I pushed the change already. I will push it for Poisson as well.

**Review by bvdmitri — APPROVED** [review]
Great work @Nimrais and @ismailsenoz ! LGTM

**Pull request merged**